utils/parts.stanza

```diff
@@ -389,6 +391,32 @@ defn string-or-unknown (json: JSON) -> String|UNKNOWN :
     (s: JNull) :
       UNKNOWN
 
+public defenum Category :
+  capacitor
+  connector
+  diode
+  fuse
+  inductor
+  microcontroller
+  oscillator
+  power_supply_chip
```

Comment:
Do we want this with `-` instead of `_`?

Reply:
Good call!

The fields within the schema itself all use underscores (to be Python-friendly) - but, here it makes sense to use Stanza idioms.